packages/repository-json-schema/src/filter-json-schema.ts

```diff
@@ -95,3 +90,24 @@ export function getWhereJsonSchemaFor(modelCtor: typeof Model): JsonSchema {
   };
   return schema;
 }
+
+/**
+ * Build a JSON schema describing the format of the "fields" object
+ * used to include or exclude properties of model instances.
+ *
+ * @param modelCtor The model constructor to build the filter schema for.
+ */
+
+export function getFieldsJsonSchemaFor(modelCtor: typeof Model): JsonSchema {
+  const schema: JsonSchema = {
+    type: 'object',
+    properties: Object.assign(
+      {},
+      ...Object.keys(modelCtor.definition.properties).map(k => ({
+        [k]: {type: 'boolean'},
+      })),
+    ),
+    additionalProperties: true,
```

Comment:
I think `additionalProperties` should be set to:
- `false` when the model is in strict mode and does not allow any additional properties (this is the default in LB4)
- `true` when the model disabled strict mode and allows any additional properties (FWIW, this is the default in LB3)

Would you mind changing your pull request accordingly?

I think you can find the relevant model setting via `modelCtor.definition.settings.strict`.

Reply:
done